include/stp/ToSat/BBNodeManagerAIG.h

```diff
@@ -30,9 +30,9 @@ THE SOFTWARE.
 #include "BBNodeAIG.h"
 #include "stp/ToSat/ToSATBase.h"
 
-#include "extlib-abc/aig.h"
-#include "extlib-abc/cnf_short.h"
-#include "extlib-abc/dar.h"
+#include "aig/aig/aig.h"
+#include "sat/cnf/cnf.h"
+#include "opt/dar/dar.h"
```

Comment:
A downside here is that we lose the context that these headers are under `abc/`, however the directory structure does not allow us to do this. We could make a symlink `include/abc -> lib/abc/src` but I don't think that will work on Windows.

We should probably ask the ABC project to restructure its headers into an `include/` directory.

Reply:
I guess that just adding comments is enough. Because ABC is highly popular project, may be they will not hope to change the structure.

Reply:
Added comments.